src/app/app.component.html

```diff
@@ -4,8 +4,8 @@
 <nav class="wim-nav hidden-print">
     <div class="container" style="width: 100%;">
         <button class="mobile-nav-toggle" (click)="toggleSidebar()"></button>
-        <a href="https://usgs.gov" class="nav-branding">
-            <img src="https://wim.usgs.gov/website-assets/usgs-logo.png" />
+        <a href="https://usgs.gov">
+            <img src="https://wim.usgs.gov/src/images/usgs.png" height="60px" width="150px" style="margin-top: 6px"/>
```

Comment:
Generally, we try not to do inline styling (in the html).

Can you add a class or id and put this in a css file instead?